test/e2e/support/reports.js

```diff
@@ -1,15 +1,15 @@
-const reporter = require('cucumberjs-allure-reporter');
-const fs = require('fs');
-const mkdirp = require('mkdirp');
+// const reporter = require('cucumberjs-allure-reporter');
+// const fs = require('fs');
+// const mkdirp = require('mkdirp');
 
-const xmlReports = `${process.cwd()}/reports/xml`;
+// const xmlReports = `${process.cwd()}/reports/xml`;
```

Comment:
Same for this commented-out line.

Reply:
i have removed the contents of the entire file